src/AdminConsole/Db/ConsoleDbContext.cs

```diff
@@ -41,6 +41,10 @@ protected override void OnModelCreating(ModelBuilder builder)
         builder.Entity<OrganizationEvent>()
             .HasKey(x => x.Id);
 
+        builder.Entity<Organization>()
```

Comment:
We will store whether magic links are enabled on the `Organization` entity, and enable it by default. This should give us also backwards compatibility without issues.